src/components/FoundationLayout/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import { Link } from "gatsby"
+import DocsSidebar from "../DocsSidebar"
+import Marked from '../Marked'
+
+interface Props {
+  title: string
+  heroText: string
+  nextDoc: any
+  permalink: string
+  sideBarData: any
+  rawMarkdownBody: string
+  pageContext: any
+}
+
+const index = ({ title, heroText, nextDoc, sideBarData, rawMarkdownBody, pageContext }: Props) => {
```

Comment:
Just to be consistent with component names in other files
```suggestion
const Index = ({ title, heroText, nextDoc, sideBarData, rawMarkdownBody, pageContext }: Props) => {
```

Reply:
Thank you, this is in the commit I just pushed.